includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -906,6 +906,32 @@ public function should_show_payment_request_button() {
 			return false;
 		}
 
+		if ( $this->is_product() && $this->get_product()->get_type() === 'variable' ) {
```

Comment:
@asumaran, should we also apply this behavior to `variable-subscription`?

Reply:
@lovo-h we should check if the fix also applies for variable subscriptions as well since we support variable subscriptions with PRBs https://github.com/woocommerce/woocommerce-gateway-stripe/blob/aa2de49012b8a434dd74e56742ef190a8fa4901c/includes/payment-methods/class-wc-stripe-payment-request.php#L560-L562

Reply:
I tested for this and it looks like this fix should also be applied to `variable-subscription`. 

I'll go ahead and update this piece of code and update the PR's notes so that it includes a test for `variable-subscription`.